components/03-modules/newsletter/_newsletter.scss

```diff
@@ -0,0 +1,94 @@
+$newsletter__font-size                : $font-size-base !default;
+$newsletter__font-size--agreements    : $font-size-small !default;
+$newsletter__field-border-radius      : 100px !default;
+$newsletter__checkbox-label-opacity   : 0.7 !default;
+$newsletter__line-height              : 1.2 !default;
+$newsletter__input-background         : #717171 !default;
+$newsletter__input-color              : $white !default;
+$newsletter__input-color--placeholder : rgba(255, 255, 255, 0.7) !default;
+$newsletter__heading-color            : $white !default;
+$newsletter__controls-height          : 32px !default;
+$newsletter__checkbox-bg-color        : $white !default;
+$newsletter__heading-gap              : $spacer--medium !default;
+
+.newsletter {
+    &__heading {
+        line-height: $newsletter__line-height;
+        margin-right: 20px;
+        color: $newsletter__heading-color;
+        font-family: $font-family-base;
+        height: auto;
+        font-size: $newsletter__font-size;
+        font-weight: $font-weight-bold;
+        text-align: left;
+        margin-bottom: $newsletter__heading-gap;
+    }
+    &__controls {
+        position: relative;
+        height: $newsletter__controls-height;
+        margin-bottom: $spacer--medium;
+    }
+    &__button {
+        position: absolute;
+        top: 0;
+        right: 0;
+        height: $newsletter__controls-height;
+        border-radius: $newsletter__field-border-radius;
+        padding-left: 21px;
+        padding-right: 21px;
+        &:hover,
+        &:focus {
+            cursor: pointer;
+            outline: none;
+        }
+    }
+    &__input {
+        margin: 0;
+    }
+    &__label {
+        @include visually-hidden;
+    }
+    &__field {
+        background: $newsletter__input-background;
+        padding: $spacer $spacer--medium;
+        margin: 0;
+        border: none;
+        border-radius: $newsletter__field-border-radius;
+        height: 32px;
+        color: $newsletter__input-color;
+        &::placeholder {
+            color: $newsletter__input-color--placeholder;
+        }
+    }
+    &__agreements {
+        margin-left: $spacer;
+    }
+    // TODO: add {{class}} for checkbox html tags
+    .checkbox__icon {
+        top: 0;
```

Comment:
I think it will be good to make a variable with that. One variable to this value and from 87 line. For now in preview it's bad aligned

Reply:
I changed for variable but only top, b/c it could be changed if we change `line-height` value, for `left` position, I left 0 without any variable